src/System.Linq.Parallel/tests/QueryOperators/SingleSingleOrDefaultTests.cs

```diff
@@ -167,6 +166,14 @@ public static void SingleOrDefault_OneMatch_Longrunning(Labeled<ParallelQuery<in
         }
 
         [Theory]
+        [MemberData(nameof(UnorderedSources.Ranges), new[] { 128 }, MemberType = typeof(UnorderedSources))]
+        public static void Single_OperationCanceledException(Labeled<ParallelQuery<int>> labeled, int count)
+        {
+            Functions.AssertEventuallyCanceled((token, canceler) => labeled.Item.WithCancellation(token).Single(x => { canceler(); return false; }));
+            Functions.AssertEventuallyCanceled((token, canceler) => labeled.Item.WithCancellation(token).SingleOrDefault(x => { canceler(); return false; }));
```

Comment:
Why in some cases are Single and SingleOrDefault combined into the same test method and in other cases they're in two different ones?  My default assumption for different named operators is that they should be in distinct test methods.